mods/ra/maps/soviet-01/soviet01.lua

```diff
@@ -25,7 +25,6 @@ JeepDemolishingBridge = function()
 	StartJeep.Move(StartJeepMovePoint.Location)
 
 	Trigger.OnIdle(StartJeep, function()
-		Trigger.ClearAll(StartJeep)
```

Comment:
I believe this one actually is required to keep this function from triggering needlessly again.